Sources/carton/Helpers/FileSystem.swift

```diff
@@ -38,4 +38,11 @@ extension FileSystem {
 
     return result
   }
+
+  func humanReadableFileSize(_ path: AbsolutePath) throws -> String {
+    precondition(isFile(path))
+
+    // FIXME: should use `UnitInformationStorage`, but it's unavailable in open-source Foundation
```

Comment:
<!--
  1 failure:  FIXMEs should be ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/carton/Helpers/FileSystem.swift;
  Line: 45;
-->
- :no_entry_sign: FIXMEs should be resolved (should use `UnitInformationSto...). (`todo`)